.buildkite/release-builds.yml

```diff
@@ -10,7 +10,7 @@ common_params:
         repo: "woocommerce/woocommerce-ios/"
   # Common environment values to use with the `env` key.
   - &common_env
-    IMAGE_ID: xcode-13.3.1
+    IMAGE_ID: xcode-13.4.1
```

Comment:
I didn't test the release build pipeline to avoid ASC issues. I think we can be reasonably sure that it works if the other pipelines do.